docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -315,3 +315,20 @@ The following migration example allows users with the `Editor` role to access th
                   - identifier: activity_log_owner
                     values: []
 ```
+
+## Update Elasticsearch schema
+
+Elasticsearch schema evolves. You need to erase its index, update the schema, and rebuild the index.
```

Comment:
```suggestion
Elasticsearch schema evolves.
You need to erase its index, update the schema, and rebuild the index.
```
How does the schema evolve, because of what?
With time, because of changes to the the content model?
One or two reasons for schema's evolution would help get into the mind of the developer who needs to perform these actions.

Reply:
@dabrt 
```suggestion
Elasticsearch schema's templates change with the addition of new features such as spellchaking.
You need to erase the index, update the schema, and rebuild the index.
```